libs/log.c

```diff
@@ -57,6 +58,12 @@ log_open(void)
 		expanded_path = expand_path(path);
 
 		path = fxstrdup(expanded_path);
+		if (strchr(expanded_path, '/') != NULL)
```

Comment:
To make the patch for $FVWM3_LOGFILE use 'starts with /' instead of 'contains /', change this line to (I haven't tested this, so hopefully my C syntax is correct):

```suggestion
		if (expanded_path[0] == '/')
```

I think it is safe to assume expanded_path is not NULL so just checking if the first character is '/' should be sufficient. This small change should make the logic starts with '/' vs contains to prepend $FVWM_USERDIR or not. So update this if you want that change.

If $FVWM_USERDIR prepends $HOME or not is a separate issue (if it even is one -- I think the logic there is currently fine) and shouldn't be part of this patch IMO.